Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -16,71 +16,85 @@
 2. When submitting to release, you are asked if the app uses the Advertising Identifier (IDFA). The answer is YES. You are then presented with three options please select as followed:
 	1. 🚫 Serve advertisements within the app
 	2. ✅ Attribute this app installation to a previously served advertisement
 	3. ✅ Attribute an action taken within this app to a previously served advertisement
 
-**Phase 5: Closure**
-<br/>	*It starts after the app is accepted by Apple and final internal approval*
+### Phase 5: Closure
+*It starts after the app is accepted by Apple and final internal approval*
 
-1. Send the build to Testflight Beta (external testing). Select the `External Testers` group.
+1. Send the build to TestFlight Beta (external testing). Select the `External Testers` group.
 1. Press `Release this version` in App Store Connect
-1. Tag the release commit and create a GitHub release. Attach the binary as an artefact to the GitHub release (if you're using the automated release command, you can find the binary in the Artifacts top section in the CI build).
+1. Create a tag named `{appname}/{version}` (e.g. `babylon/4.1.0`) on the release commit and create a GitHub release for that new tag
+  - Make sure you create separate tags (and GitHub releases) for each app released on the AppStore (eg. Babylon 4.1.0 and Telus 4.1.0 would each have their own `babylon/4.1.0` and `telus/4.1.0` tags)
+  - Set the body of the GitHub release to the content of the Release Notes for the app
+  - Attach the binary as an artefact to the GitHub release (if you're using the automated release command, you can find the binary in the Artifacts top section in the CI build).
 1. Merge the changes back to develop.
 1. Update the [release calendar](#release-calendar)
 
-### 3. SDK Release
+## 3. SDK Release
 
 1. Ask SDK team (#sdk_squad) about the SDK version number.
-2. Follow the [Internal SDK Release Process](https://engineering.ops.babylontech.co.uk/docs/cicd-deployments/#mobile-sdk-releases-ios-android) to open a CR ticket on the CRP board.
-3. Cut a release branch for the SDK from the app release branch (eg. `sdk/0.5.0`)
-4. Create PR and udpate the SDK changelog `SDK/CHANGELOG.md` to add the release version and date
-5. Trigger a hockey build from that branch using its command (eg. `/fastlane distribute_sdk_v2 version:0.5.0 branch:release/sdk/0.5.0`) in `#ios-build`.
-6. Update the Sample app to point to the latest SDK release
+1. Cut a release branch for the SDK from the app release branch, using the `sdk/{version}` naming convention (eg. `sdk/0.5.0`)
+1. Create a CRP ticket by triggering its command (eg. `/crp ios branch:release/sdk/0.5.0`) in Slack
+  - This will create the CRP ticket for the SDK, only including in the CHANGELOG field of the CRP the commits messages containing `[SDK-xxx]` or `#SDK` – filtering out the other commits, that are considered app-only changes if not containing those tags
```

Comment:
ℹ️ There are future plans to enforce that convention via Danger – so that any PR which contains changes to any files in the SDK/ folder will be required to have the `#SDK` tag in its title to ensure the squashed commit will be listed in the SDK CRP ticket. I'll do a PR to add that Danger rule soon.

Reply:
What about release version and date ? Will that be added to `SDK/CHANGELOG.md` when this command is triggered?

Reply:
the `SDK/CHANGELOG.md` is totally untouched by the CRP command, which is only based on the commit list, even for the SDK, and not the `CHANGELOG.md` file.

The `CHANGELOG.md` file will still need to be filled manually for now (since it's a file targeted for partners using the SDK, potentially worded in a more human-readable and filled manually). Besides, it should only be updated with the release date once the release is ready, so once QA has signed off the release, not at the time the release branch is cut, I guess, in case a change had to be introduced during a fix for a bug reported by QA.

What we can imagine doing though is have a `Danger` that will check that if the PR is for merging a branch named `release/sdk/{version}`, ensure that you have a `## {version}` section in the `SDK/CHANGELOG.md` file in that PR. That way we could check those at the time we're ready to merge the release (and thus the date should match), as opposed to do that when we cut off the release/sdk/{version} branch at the beginning of the release process. wdyt?

Reply:
Oh . I just noticed that the step was not removed. It's listed below. Thanks

Reply:
This will be fairly important imo, since there are quite a few SDK changes that are done by the individual squads rather than the SDK squad itself